### PR TITLE
enable ensure switch on streams-available/* files

### PR DESCRIPTION
**Pull request by aleksmark**
Issue opened https://github.com/voxpupuli/puppet-nginx/issues/1304

### CODE REVIEW

**Comment by bastelfreak** [comment]
Hi @aleksmark,
can you have a look at the used email address in the commit? It isn't associated with your github account. Also please check the failed travis jobs.

File: manifests/resource/streamhost.pp [hunk 1]
```diff
@@ -103,6 +103,7 @@
   }
 
   concat { $config_file:
+    ensure => $ensure,
```
Comment (bastelfreak) [inline manifests/resource/streamhost.pp]: the alignment isn't correct here:
```suggestion
    ensure  => $ensure,
```

**Comment by aleksmark** [comment]
@bastelfreak closing this one and opening https://github.com/voxpupuli/puppet-nginx/pull/1306 with email fixed on commit and comment addressed

**Comment by aleksmark** [comment]
@bastelfreak  will check why the tests are failing

**Pull request closed without merging**